app/views/support_interface/application_forms/courses/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title, title_with_error_prefix("Select a course to add to #{@pick_course.applicant_name}‘s application", @pick_course.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_search_course_new_path(course_code: @pick_course.course_code)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if @pick_course.course_options.present? %>
+      <%= form_with model: @pick_course, url: support_interface_application_form_create_course_path(course_code: @pick_course.course_code) do |f| %>
+        <%= f.govuk_error_summary %>
+        <%= f.govuk_radio_buttons_fieldset :course_option_id, legend:  { text: "Which course should be added to the application?", size: "m" } do %>
+          <% @pick_course.course_options.each do |co| %>
+            <%= f.govuk_radio_button :course_option_id, co.course_option_id, label: { text: "#{co.course_name} (#{co.course_code}) - #{co.site_name}" } %>
```

Comment:
I am wondering whether we should add provider_name to the struct and show it here. I have run into courses with the same name and course_code. It might be overkill as they won't share a site though.

Reply:
Yep cool - I will add that in